packages/react/.storybook/addon-fathom/register.js

```diff
@@ -0,0 +1,27 @@
+import { window } from 'global';
+import { addons } from '@storybook/addons';
+import { STORY_CHANGED } from '@storybook/core-events';
+
+addons.register('fathom', api => {
+  (function(f, a, t, h, o, m) {
+    a[h] =
+      a[h] ||
+      function() {
+        (a[h].q = a[h].q || []).push(arguments);
+      };
+    (o = f.createElement('script')), (m = f.getElementsByTagName('script')[0]);
+    o.async = 1;
+    o.src = t;
+    o.id = 'fathom-script';
+    m.parentNode.insertBefore(o, m);
+  })(document, window, 'https://cdn.usefathom.com/tracker.js', 'fathom');
+  fathom('set', 'siteId', 'LLBKOSZK');
+
+  api.on(STORY_CHANGED, () => {
+    const pathString = api.getUrlState().path;
+
+    if (typeof window.fathom === 'function') {
+      window.fathom('trackPageview', { path: pathString });
```

Comment:
```suggestion
      window.fathom('track', { path: pathString }, 'pageview');
```

Seems like this should work based on: https://github.com/usefathom/fathom/issues/305#issuecomment-589785554 ?

Reply:
I lied, doesn't work 🤔

Reply:
@joshblack Since you're doing the call manually, can I confirm you don't have fathom('set', 'spa', 'pushstate'); in your code)? I ask because that will stop the path from being overriden

Reply:
@JackEllis thanks for chiming in! Went ahead and removed that code snippet you shared from the file 👀